packages/contracts-bedrock/test/L2/L1BlockInterop.t.sol

```diff
@@ -1,234 +1,188 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.15;
 
-import { Test } from "forge-std/Test.sol";
-import { Encoding } from "src/libraries/Encoding.sol";
-import { L1BlockInterop, DependencySetSizeMismatch, NotDepositor } from "src/L2/L1BlockInterop.sol";
-
-contract L1BlockInteropTest is Test {
-    L1BlockInterop l1Block;
-    address depositor;
-
-    function setUp() public {
-        l1Block = new L1BlockInterop();
-        depositor = l1Block.DEPOSITOR_ACCOUNT();
-    }
-
-    /// @dev Tests that setL1BlockValuesInterop updates the values appropriately.
-    function testFuzz_setL1BlockValuesInterop_succeeds(
-        uint32 _baseFeeScalar,
-        uint32 _blobBaseFeeScalar,
-        uint64 _sequenceNumber,
-        uint64 _timestamp,
-        uint64 _number,
-        uint256 _baseFee,
-        uint256 _blobBaseFee,
-        bytes32 _hash,
-        bytes32 _batcherHash,
-        uint256[] calldata _dependencySet
-    )
-        external
-    {
-        vm.assume(_dependencySet.length <= type(uint8).max);
-        vm.assume(uint160(uint256(_batcherHash)) == uint256(_batcherHash));
-
-        bytes memory functionCallDataPacked = Encoding.encodeSetL1BlockValuesInterop({
-            _baseFeeScalar: _baseFeeScalar,
-            _blobBaseFeeScalar: _blobBaseFeeScalar,
-            _sequenceNumber: _sequenceNumber,
-            _timestamp: _timestamp,
-            _number: _number,
-            _baseFee: _baseFee,
-            _blobBaseFee: _blobBaseFee,
-            _hash: _hash,
-            _batcherHash: _batcherHash,
-            _dependencySet: _dependencySet
-        });
-
-        vm.prank(depositor);
-        (bool success,) = address(l1Block).call(functionCallDataPacked);
-        assertTrue(success, "Function call failed");
-
-        assertEq(l1Block.baseFeeScalar(), _baseFeeScalar);
-        assertEq(l1Block.blobBaseFeeScalar(), _blobBaseFeeScalar);
-        assertEq(l1Block.sequenceNumber(), _sequenceNumber);
-        assertEq(l1Block.timestamp(), _timestamp);
-        assertEq(l1Block.number(), _number);
-        assertEq(l1Block.basefee(), _baseFee);
-        assertEq(l1Block.blobBaseFee(), _blobBaseFee);
-        assertEq(l1Block.hash(), _hash);
-        assertEq(l1Block.batcherHash(), _batcherHash);
-        assertEq(l1Block.dependencySetSize(), _dependencySet.length);
+import { CommonTest } from "test/setup/CommonTest.sol";
+
+// Target contract dependencies
+import { L1BlockInterop, ConfigType } from "src/L2/L1BlockInterop.sol";
+import "src/libraries/L1BlockErrors.sol";
+
+contract L1BlockInteropTest is CommonTest {
```

Comment:
Are you sure this is in the right PR? These tests are for `L1Block` but this PR is for `OptimismPortal`